release-please-config.json

```diff
@@ -0,0 +1,23 @@
+{
+    "bootstrap-sha": "9f719b9fce8b79e75b8f0021527cb09b221368a5",
+    "bump-minor-pre-major": true,
+    "bump-patch-for-minor-pre-major": true,
+    "always-link-local": false,
+    "separate-pull-requests": false,
```

Comment:
Does it mean we will have one Release PR for all packages? What if we just want to release a single package?

Reply:
yes, that's the best configuration recommended for monorepos. I tried option to have separate release PR per package, the problem is that main `package-lock.json` is in root of the project and does not belong to each individual package. So the tool starts to open separate PR for monorepo root, and individual PRs don't include update to `package-lock`. It can be problematic to configure per package, so I suggest to start with 1 PR for all packages.

Reply:
is this behaviour and the reasoning documented somewhere already